pkg/clusterid/override_parser_dev.go

```diff
@@ -0,0 +1,16 @@
+//go:build !release
+
+package clusterid
+
+import (
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var mu sync.Mutex
```

Comment:
If you want to keep using `parserWrapper` maybe make this a property of the wrapper struct?

Reply:
I've removed the wrapper 🙂